_positions/product-manager.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Product Manager
+permalink: pay-grades/product-manager/
+team: 18F
+active: true
+---
+**GS Grade Level:** 13
```

Comment:
@hestherchang This is the new spot to make changes.

That `pay-grades` part of the permalink is a holdover; I'll update the URL to `positions/product-manager/`.